modules/packages/BLAS.chpl

```diff
@@ -197,7 +197,7 @@ module BLAS {
           alpha1, A, _ldA, B, _ldB, beta1, C,_ldC);
       }
       otherwise {
-        halt("Unknown type in gemm");
+        compilerError("Unknown type in gemm");
```

Comment:
Though a pain, it might be good to print out the unknown type's name, like this:
```chpl
compilerError("Unknown type in gemm: ", eltType:string);
```

(I think that's the correct code)